tasks/setup-Debian.yml

```diff
@@ -26,12 +26,12 @@
       state: present
   when: ansible_distribution_release == 'stretch' and 'openjdk-11-jdk' in java_packages
 
-- name: Ensure Java is installed (stretch + jdk11).
+- name: Ensure Java is installed and is the latest maintenance version (stretch + jdk11).
```

Comment:
Just note that this fix only applies to `stretch`. Shall we also use `latest` on other jdk versions and distrib versions?

Reply:
(`inte-mia` is on `buster` for example)

Reply:
@gsaint good question, I can add it for all versions

Reply:
@gsaint fixed, thx for the feedback